default/python/AI/AIstate.py

```diff
@@ -467,14 +483,27 @@ def update_system_status(self):
             mob_rating = CombatRatingsAI.combine_ratings_list(mob_ratings)
             if fleetsLostBySystem.get(sys_id, []):
                 lost_fleet_rating = CombatRatingsAI.combine_ratings_list(fleetsLostBySystem[sys_id])
-            if not system or partial_vis_turn == -9999:  # under current visibility rules should not be possible to have any losses or other info here, but just in case...
+            # under current visibility rules should not be possible to have any losses or other info here,
+            # but just in case...
+            if not system or partial_vis_turn == -9999:
                 if verbose:
-                    print "Have never had partial vis for system %d ( %s ) -- basing threat assessment on old info and lost ships" % (sys_id, sys_status.get('name', "name unknown"))
+                    print ("Have never had partial vis for system %d ( %s ) "
+                           "-- basing threat assessment on old info and lost ships") % (
+                        sys_id, sys_status.get('name', "name unknown"))
                 sys_status.setdefault('local_fleet_threats', set())
                 sys_status['planetThreat'] = 0
-                sys_status['fleetThreat'] = int(max(CombatRatingsAI.combine_ratings(enemy_rating, mob_rating), 0.98 * sys_status.get('fleetThreat', 0), 1.1 * lost_fleet_rating - monster_rating))
-                sys_status['monsterThreat'] = int(max(monster_rating, 0.98 * sys_status.get('monsterThreat', 0), 1.1 * lost_fleet_rating - enemy_rating - mob_rating))
-                sys_status['enemy_threat'] = int(max(enemy_rating, 0.98 * sys_status.get('enemy_threat', 0), 1.1 * lost_fleet_rating - monster_rating - mob_rating))
+                sys_status['fleetThreat'] = int(max(
+                    CombatRatingsAI.combine_ratings(enemy_rating, mob_rating),
+                    0.98 * sys_status.get('fleetThreat', 0),
+                    1.1 * lost_fleet_rating - monster_rating)
+                )
```

Comment:
I usualy put closing on same line, if args starts on same line as open:
```
foo(arg1, arg2,
       arg3)
```
https://github.com/Cjkjvfnby/freeorion/blob/master/default/python/AI/AIstate.py#L851

In case when I put arguments to new line: 

```
foo(
  arg1, arg2,  arg3
)
```
https://github.com/Cjkjvfnby/freeorion/blob/master/default/python/AI/AIDependencies.py#L59
https://github.com/Cjkjvfnby/freeorion/blob/master/default/python/AI/AIstate.py#L764